vanilla-javascript-demo/README.md

```diff
@@ -0,0 +1,68 @@
+# Using ChemDraw JS with Vanilla JavaScript
+
+Example of how to use the ChemDraw JS library with vanilla JavaScript and HTML5.
+
+This example demonstrates how to load the ChemDraw JS library when installed as an npm package. It is based on the example in the __ChemDraw JS Developer's Guide__. For a project without npm the same code can be used, just change the URL to the library as appropriate.
+
+For this demo, copy the license into /dependencies.
+
+To install and run the demo using a terminal starting at the repository root directory:
+
+- cd to the project directory: `cd vanilla-javascript-demo`
+- Install project dependencies using npm: `npm ci`
+- Start the development server: `npm start`
+
+# Getting started with ChemDraw JS and Parcel
+
+## Installing ChemDraw JS
+
+You should have received three files as part of your ChemDraw JS subscription:
+
+1. An npm package as a .tgz file
+2. A license as an XML file
+3. A ChemDraw Web Service (CDWS) docker image
+
+For this demo, copy the license and npm package into /dependencies. The npm package will need to be renamed as `chemdraw-js.tgz`.
+
+For full functionality, the web service will need to be run using Docker. Map port 80 in the Docker image to 8080 on your machine, e.g.:
+```
+docker load --input chemdraw-web-service-2.2.0-develop-1150.tar.gz
+docker run -d -p 8080:80 pkiinformatics/chemdraw-web-service:2.2.0-develop-1150
+```
+
+## Bundling with Parcel
+Parcel is billed as a "zero configuration web application bundler". This zero configuration philosophy causes issues with ChemDraw JS, since any attempt to reference the library from code will cause it to attempt to bundle the library which breaks the internal dynamic loading of assets.
+
+To work around this problem ensure that the script tag in the html entry point of the application uses a qualified http location to the library. By prefixing the URL with http, parcel will ignore the URL. In our example you can see we use:
```

Comment:
Do we really need all this commentary? If using Parcel has made this more complicated then please remove Parcel. The whole point of this example is that it is as simple as possible.

Reply:
> Do we really need all this commentary? If using Parcel has made this more complicated then please remove Parcel. The whole point of this example is that it is as simple as possible.

Redundancy deleted